internal/pkg/cli/flag.go

```diff
@@ -29,23 +29,24 @@ const (
 	manifestFlag   = "manifest"
 
 	// Command specific flags.
-	dockerFileFlag        = "dockerfile"
-	dockerFileContextFlag = "build-context"
-	imageTagFlag          = "tag"
-	resourceTagsFlag      = "resource-tags"
-	stackOutputDirFlag    = "output-dir"
-	uploadAssetsFlag      = "upload-assets"
-	limitFlag             = "limit"
-	lastFlag              = "last"
-	followFlag            = "follow"
-	sinceFlag             = "since"
-	startTimeFlag         = "start-time"
-	endTimeFlag           = "end-time"
-	tasksFlag             = "tasks"
-	logGroupFlag          = "log-group"
-	prodEnvFlag           = "prod"
-	deployFlag            = "deploy"
-	resourcesFlag         = "resources"
+	dockerFileFlag          = "dockerfile"
+	dockerFileContextFlag   = "build-context"
+	imageTagFlag            = "tag"
+	resourceTagsFlag        = "resource-tags"
+	stackOutputDirFlag      = "output-dir"
+	uploadAssetsFlag        = "upload-assets"
+	limitFlag               = "limit"
+	lastFlag                = "last"
+	followFlag              = "follow"
+	previousStoppedTaskFlag = "previous-stopped-task"
```

Comment:
can we rename the flag to just `-p, --previous` (this way we can reuse it maybe across commands)

Reply:
I was just wondering when we use shortcut flags like -p? 
Looks like we use them only when they are mandatory by the command and this flag is not mandatory anywhere that is why I did not create shortcut. Is my understanding correct?

Reply:
Your understanding is correct! More specifically:
> Required flags must have a short name, only provide a short name to an optional flag **if you think it will be frequently used.**
> https://github.com/aws/copilot-cli/blob/mainline/STYLE_GUIDE.md#flags

I feel like this is a really neat flag 🌈  that would be used frequently to troubleshoot so `-p` is warranted

Reply:
Got it.